Api/ResourceResolver.php

```diff
@@ -21,7 +21,7 @@
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Samuel ROZE <samuel.roze@gmail.com>
  */
-trait ResourceResolverTrait
```

Comment:
Ok :+1: I'm convinced for this one. But traits are horizontal composition, not copy/paste.